README-SL.md

```diff
@@ -108,18 +107,17 @@ Povezave, ki neposredno usmerjajo na primerne `issues`, s katerim lahko prispeva
 - [Prvič z odprtokodnimi programi](https://www.nearform.com/blog/getting-into-open-source-for-the-first-time/) by [@mcdonnelldean](https://github.com/mcdonnelldean)
-- [Kako prispevati k odprtokodnim projektom](https://opensource.guide/how-to-contribute/) by [@GitHub](https://github.com/github)
-- [Kako najti napake v svoji kodi](https://8thlight.com/blog/doug-bradbury/2016/06/29/how-to-find-bug-in-your-code.html) by [@dougbradbury](https://twitter.com/dougbradbury)
-- [Obvladovanje Markdowna](https://guides.github.com/features/mastering-markdown/) by [@GitHub](https://github.com/github)
-- [Prva misija: stran za sodelavce](https://medium.com/@forCrowd/first-mission-contributors-page-df24e6e70705#.2v2g0no29) by [@forCrowd](https://github.com/forCrowd)
+- [Kako prispevati k odprtokodnim projektom](https://opensource.guide/how-to-contribute/) by [@GitHub](https://github.com/github/opensource.guide)
+- [Kako najti napake v svoji kodi](https://8thlight.com/insights/how-to-find-a-bug-in-your-code) by [@dougbradbury](https://twitter.com/dougbradbury)
+- [Obvladovanje Markdowna](https://docs.github.com/features/mastering-markdown/) by [@GitHub](https://github.com/github/docs)
+- [Prva misija: stran za sodelavce](https://forcrowd.medium.com/first-mission-contributors-page-df24e6e70705#.2v2g0no29) by [@forCrowd](https://github.com/forCrowd)
 - [Kako narediti svoj prvi odprtokodni prispevek v samo 5 minutah](https://www.freecodecamp.org/news/how-to-make-your-first-open-source-contribution-in-just-5-minutes-aaad1fc59c9a/) by [@roshanjossey](https://github.com/Roshanjossey/)
 - [Pravkar sem dobil svojo brezplačno majico Hacktoberfest. Tukaj je hiter način, kako lahko dobite svojo.](https://www.freecodecamp.org/news/i-just-got-my-free-hacktoberfest-shirt-heres-a-quick-way-you-can-get-yours-fa78d6e24307/) by [@quincylarson](https://www.freecodecamp.org/news/author/quincylarson/)
 - [Grenak vodnik za odprto kodo](https://medium.com/codezillas/a-bitter-guide-to-open-source-a8e3b6a3c1c4) by [@ken_wheeler](https://medium.com/@ken_wheeler)
 - [Vodnik po korakih za mlajše razvijalce, kako prvič prispevati k odprtokodni različici](https://hackernoon.com/contributing-to-open-source-the-sharks-are-photoshopped-47e22db1ab86) by [@LetaKeane](https://hackernoon.com/u/letakeane)
 - [Naučite se Git in GitHub korak za korakom (v sistemu Windows)](https://medium.com/illumination/path-to-learning-git-and-github-be93518e06dc) by [@ows-ali](https://ows-ali.medium.com/)
 - [Zakaj odprtokodna in kako?](https://careerkarma.com/blog/open-source-projects-for-beginners/) by [@james-gallagher](https://careerkarma.com/blog/author/jamesgallagher/)
 - [Kako začeti z odprto kodo - avtor Sayan Chowdhury](https://www.hackerearth.com/getstarted-opensource/)
-- [Katerim odprtokodnim projektom prispevati?](https://kentcdodds.com/blog/what-open-source-project-should-i-contribute-to/) by [@kentcdodds](https://github.com/kentcdodds)
```

Comment:
This can be kept and added back in.

Reply:
done https://github.com/freeCodeCamp/how-to-contribute-to-open-source/commit/73dee199115c07bd01eebf021e85d48d048ab98c